ffi/ffi.cpp

```diff
@@ -0,0 +1,256 @@
+#include "ffi.h"
+#include "ffi_serialization.hpp"
+#include <libff/algebra/curves/bls12_377/bls12_377_pp.hpp>
+#include <libff/algebra/curves/bw6_761/bw6_761_pp.hpp>
+
+namespace libff
+{
```

Comment:
We do not break after the namespace in this code base.

Reply:
removed the break

Reply:
This also needs to be done in the `serialization` files. Then we're ready to go.

Reply:
Should be addressed in latest commit.